### PR TITLE
Feature/add temp endpoint

### PR DESCRIPTION
**Pull request by alaeddineallouchi**
please check because i have a can't automatically merge message

### CODE REVIEW

File: server2.py [hunk 1]
```diff
@@ -7,5 +7,22 @@
...
+def smoke() : 
+    return"no smoke "
+
+@app.route("/movement")
+def movement() : 
+    return"yes or no "
+
 if (__name__=="__main__") : 
-    app.run()
+    app.run(host="0.0.0.0")
```
Comment (lwiss) [inline server2.py:28]: Explicitly add port

File: server2.py [hunk 2]
```diff
@@ -7,5 +7,22 @@
...
+def humidity() : 
+    return"yes or no "
+
+@app.route("/smoke")
+def smoke() : 
+    return"no smoke "
+
+@app.route("/movement")
+def movement() : 
+    return"yes or no "
```
Comment (lwiss) [inline server2.py:25]: Coding style : add space after return

File: server2.py [hunk 3]
```diff
@@ -7,5 +7,22 @@
...
     return"Hello_World"
 
+
+@app.route("/temp")
+def temp() : 
+    return"temp is 20 deg"
+
+@app.route("/humidity")
+def humidity() : 
+    return"yes or no "
```
Comment (lwiss) [inline server2.py]:
```suggestion
    return "yes or no "
```

File: server2.py [hunk 4]
```diff
@@ -7,5 +7,22 @@
 def welcome() : 
     return"Hello_World"
 
+
+@app.route("/temp")
+def temp() : 
+    return"temp is 20 deg"
```
Comment (lwiss) [inline server2.py]:
```suggestion
    return "temp is 20 deg"
```

File: server2.py [hunk 5]
```diff
@@ -7,5 +7,22 @@
...
+    return"yes or no "
+
+@app.route("/smoke")
+def smoke() : 
+    return"no smoke "
+
+@app.route("/movement")
+def movement() : 
+    return"yes or no "
+
```
Comment (lwiss) [inline server2.py:26]: Add extra empty line

File: server2.py (hunk 1 above)
Comment (alaeddineallouchi) [reply server2.py:28]: how can i add a port ?

Comment (lwiss) [reply server2.py:28]:
```suggestion
    app.run(host="0.0.0.0", port=5001)
```

**Pull request merged**